action.yml

```diff
@@ -41,7 +41,7 @@ runs:
       shell: bash
 
     - name: Run script
-      run: python ${{ github.action_path }}/job_summary.py
+      run: INPUT_DEPENDENCIES_REPORT_PATH=${{inputs.dependencies-report-path}} INPUT_CODE_REPORT_PATH=${{inputs.code-report-path}} python ${{ github.action_path }}/job_summary.py
```

Comment:
Accept the variables as command line arguments instead. Use default file names if no value is supplied.

```
python job_summary.py <deps json here> <code json here>
```